src/constrained-input/tests/unit/ConstrainedInput.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import harness from '@dojo/framework/testing/harness';
+
+import ConstrainedInput from '../..';
+import { tsx, create } from '@dojo/framework/core/vdom';
+import TextInput from '../../../text-input';
+import validation from '../../../middleware/validation';
+
+const { registerSuite } = intern.getInterface('object');
```

Comment:
Do we not use the `bdd` interfaces for widgets? I bet it's a mix unfortunately, I think we prefer the `bdd` template... we should certainly pick one and stick to it throughout the repo.

Reply:
Soooo, yeah I prefer the bdd interface, but the other widgets I looked at do it this way. If we're ok using bdd i'm happy to switch it over...